DiffMatchPatch.zig

```diff
@@ -1208,15 +1365,12 @@ fn diffCleanupSemanticScore(one: []const u8, two: []const u8) usize {
     return 0;
 }
 
-// Define some regex patterns for matching boundaries.
-// private Regex BLANKLINEEND = new Regex("\\n\\r?\\n\\Z");
-// \n\n
-// \n\r\n
-// private Regex BLANKLINESTART = new Regex("\\A\\r?\\n\\r?\\n");
-// \n\n
-// \r\n\n
-// \n\r\n
-// \r\n\r\n
+inline fn boolInt(b: bool) usize {
+    if (b)
+        return 1
+    else
+        return 0;
+}
```

Comment:
I would write like this.  
```zig
fn boolInt(b: bool) u8 {
    return @intFromBool(b);
}
```
This is the same as writing `@as(u8, @intFromBool(b))`

Reply:
I'd guess that the compiler would emit the same code either way, but yeah that's cleaner.

Reply:
Yes code is the same in all modes except Debug: https://godbolt.org/z/ovvP6nen7